bundlewrap/cmdline/parser.py

```diff
@@ -23,6 +23,25 @@
 from .verify import bw_verify
 from .zen import bw_zen
 
+try:
+    from argcomplete import autocomplete
+
+    bash_completion = True
+except ImportError:
+    bash_completion = False
+
+
+class TargetCompleter:
+    """Completer for bw targets, which can be used by argcomplete"""
+
+    def __call__(self, prefix, **kwargs):
+        compl_file = environ.get('BW_TARGET_COMPLETION_FILE')
```

Comment:
I'd prefer if we had a fixed file name in the repository root like we already have `.secrets.cfg` and `.bw_debug_history`.